cmd/convox/rack.go

```diff
@@ -205,6 +205,27 @@ func cmdRackParamsSet(c *cli.Context) error {
 		params[parts[0]] = parts[1]
 	}
 
+	existing, err := rackClient(c).ListParameters(system.Name)
+	if err != nil {
+		return stdcli.Error(err)
+	}
+
+	hasChange := false
+	for k, v := range params {
+		res, ok := existing[k]
+		if !ok {
+			hasChange = true
+			break
+		} else if res != v {
+			hasChange = true
+			break
+		}
+	}
+
+	if !hasChange {
```

Comment:
This is specifically in the `rack params set` command so I thought that would be the only way to update the stack here?

The goal I was assigned was to validate the request and if it wouldn't make a change don't make a request to the rack and trigger failure requests